tests/load-tests.php

```diff
@@ -110,25 +110,53 @@ function( $module_settings ) {
 		);
 		$this->assertSame( $expected_active_modules, $active_modules );
 
-		// Assert that option updates affect the active modules correctly.
+		// Assert that dummy module option doesn't show in active modules.
 		$new_value = array(
 			'inactive-module' => array( 'enabled' => false ),
 			'active-module'   => array( 'enabled' => true ),
 		);
 		update_option( PERFLAB_MODULES_SETTING, $new_value );
 		$active_modules = perflab_get_active_modules();
-		$this->assertSame( array( 'active-module' ), $active_modules );
+		$this->assertSame( array(), $active_modules );
+
+		// Assert that it only allow existing modules.
```

Comment:
```suggestion
		// Assert that only allows existing modules.
```

Reply:
Thanks for the feedback! Text changes updated.